### DISCUSSION TITLE
Certificate validation not working

### DISCUSSION THREAD

**Discussion by rajeevsathish** [Sunbird-Lern/Community#115] [Issues] [answered]
Post a user consumes a course and completes the consumption the user is provided with a certificate this functionality is working fine. And the user is able to download the certificate but once the certificate QR code is scanned the user gets a URL to verify the certificate
like https://<domain>/certs/1-cf11c2ea-7ff0-41c8-b053-9ef7bd38803f?t=URL&entity=TrainingCertificate this is not working giving a error page like below
<img width="1476" alt="Screenshot 2023-05-12 at 3 03 10 PM" src="https://github.com/Sunbird-Lern/Community/assets/5653717/8b0a82f6-b961-448d-9ba9-7df340c4daba">
<img width="1382" alt="Screenshot 2023-05-12 at 3 03 24 PM" src="https://github.com/Sunbird-Lern/Community/assets/5653717/e592ce46-df33-457d-a036-7aafea399b62">
CC: @reshmi-nair @vinukumar-vs

**Comment by reshmi-nair** [comment, marked as answer]
@rajeevsathish This is because the public key configuration in RC for certificate is not proper. Please update the correct public key to make this work.
